<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminpanel</class>
 <widget class="QWidget" name="adminpanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1303</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>441</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Панель админа</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnIndTrA">
      <property name="text">
       <string>Индивидуальные  тренировки</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnTrainerA">
      <property name="text">
       <string>Тренеры</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSectionA">
      <property name="text">
       <string>Секции</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnVisitingA">
      <property name="text">
       <string>Посещение</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnStudentA">
      <property name="text">
       <string>Ученики</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPlaceA">
      <property name="text">
       <string>Место проведения</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pbBack">
      <property name="text">
       <string>Сменить пользователя</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
